src/pages/graphql/payment-services/minicart.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Payment Services product page workflow
+description: Learn how Adobe Commerce uses GraphQL to process orders with Payment Services as a payment method from the product page.
+keywords:
+  - GraphQL
+  - Payments
+---
+
+# Payment services product page workflow
+
+These steps describe the additional flow of requests and responses required to place an order after [adding a product to your cart](../tutorials/checkout/add-product-to-cart.md) with the [Payment Services](https://experienceleague.adobe.com/docs/commerce-merchant-services/payment-services/guide-overview.html) solution enabled.
+
+## Product page workflow
+
+![Payment Services sequence diagram](../../_images/graphql/payment-services-minicart.svg)
+
+1. Run the [`getPaymentConfig`](../schema/checkout/queries/get-payment-config.md) query to fetch the payment configuration needed to render details about PayPal components, such as hosted fields, smart buttons, and Apple Pay.
+
+1. Commerce returns payment configuration information.
```

Comment:
I noticed in some steps we use Commerce and in some others we use Adobe Commerce, might be better to standardarize this